### DISCUSSION TITLE
Bi-Weekly Community Meeting, Saturday, June 29, 2024

### DISCUSSION THREAD

**Discussion by wrashi** [GothenburgBitFactory/taskwarrior#3504] [Announcements]
Everyone is invited to the next TaskWarrior community bi-weekly meeting!  
Want to get a sense of what to expect? Watch the recent meeting videos here…

- [June 15](https://www.youtube.com/watch?v=qI9P1L8fg0s)
- [June 1](https://youtu.be/hZQG8jDn24I)
- [May 18](https://www.youtube.com/watch?v=r5G9yiDrgkg)

Our next meeting is 12:00 UTC Saturday, June 29, 2024 ([check that in your time zone @ timeanddate.com](https://www.timeanddate.com/worldclock/meetingtime.html?month=6&day=15&year=2024&p1=5&p2=3904&p3=3918&p4=3921&iv=0)). You can [add this event to your calendar](https://calendar.google.com/calendar/event?action=TEMPLATE&tmeid=N3ZvMWFkNGFsMjJtMzV2ODFybTRkOTNhaWogd2VzbGV5QG13cGVubmVyLmNvbQ&tmsrc=wesley%40mwpenner.com).

Google Meet joining info
Video call link: https://meet.google.com/tns-suqq-iaj
Or dial: ‪(CA) +1 289-323-1423‬ PIN: ‪830 535 232‬#
More phone numbers: https://tel.meet/tns-suqq-iaj?pin=8726596308824

The formal agenda portion of the meeting runs ~30 minutes. The room will stay open for anyone who wants to hang out or go deeper into any of the topics from the agenda.

We're also looking for anyone interested in showing the community how you use TaskWarrior.

**Comment by wrashi** [comment]
Tomorrow is our bi-weekly meeting. Looking forward to seeing everyone!
We still have an open slot for anyone wanting to show off how they use TaskWarrior…

**Reply by wrashi** [reply]
A big thank you to everyone who got involved in our community meeting on Saturday! It was another terrific gathering. The recording is a bit strange, though, and that's my fault for not being completely awake to realize I had the settings wrong. Hope you can enjoy it anyway!

https://youtu.be/M8BRQJfRgoE

**Comment by djmitche** [comment]
We've had some interesting conversations during these meetings! I want to emphasize that there is no requirement to attend -- even if you just use TW and noticed this discussion, we'd love to hear from you! Admittedly it is pretty early on a Saturday for us N & S Americans.
